libs/community/langchain_community/utilities/pebblo.py

```diff
@@ -236,6 +236,27 @@ def get_loader_full_path(loader: BaseLoader) -> str:
             location = "in-memory"
         elif isinstance(loader, NotionDBLoader):
             location = f"notiondb://{loader.database_id}"
+        elif loader.__class__.__name__ == "GoogleDriveLoader":
```

Comment:
why is this checking the class name and not `isinstance`?

Reply:
It was to avoid the unnecessary import of GoogleDriveLoader, I am planning to remove other imports and check the names for them too.

Also, particularly in case of GoogleDriveLoader, they are present in langchain_community and langchain_google both the packages.